cadccutout/README.rst

```diff
@@ -63,6 +69,44 @@ Perform a cutout from an input stream from an HTTP request.
        with open(output_file, 'ab+') as output_writer:
            test_subject.cutout(input_stream, output_writer, cutout_region_string, 'FITS')
 
+
+Command Line Access
+-------------------
+
+The executable ``cadccutout`` is installed by default, or the module can be run using ``python -m cadccutout``.
+
+Running
+~~~~~~~
+
+``cadccutout -d --infile path/to/source.fits --outfile path/to/output.fits [100:400]``
+
+``cadccutout -d --infile path/to/source.fits --outfile path/to/output.fits "CIRCLE=10 60 0.5"``
+
+
+Running in Docker
+~~~~~~~~~~~~~~~~~
+
+The provided `Dockerfile`_ can be used to build an image based on the desired
+Python version.
+
+Build an image for Python 2.7:
+
+``docker build --build-arg PYTHON_VERSION=2.7 -t opencadc/cadccutout:2.7-alpine .``
+
+Then execute it (``/usr/src/data`` is the location of the source files).  This will send the output to standard out:
+
+``docker run --rm -v $(pwd):/usr/src/data opencadc/cadccutout:2.7-alpine cadccutout --infile /usr/src/data/myfile.fits [100:400]``
+
```

Comment:
Grumble...

You think this:
`--mount type=bind,source=$(pwd),target=/usr/src/data`

is easier to remember than
`-v $(pwd):/usr/src/data`?

I'll put both down, but let it be known that `-v` is used much more widely.  

Since I'm putting both in, so do you... :)